### PR TITLE
fix(Upload): fix the order of upload doesn't correspond to the order of selected images

### PR DESCRIPTION
**Pull request by heiyu4585**
https://github.com/ant-design/ant-design-mobile/issues/5970

### CODE REVIEW

**Comment by afc163** [comment]
补个用例？

**Comment by heiyu4585** [comment]
> 补个用例？

done

File: src/components/image-uploader/tests/image-uploader.test.tsx [hunk 1]
```diff
@@ -164,6 +164,53 @@ describe('ImageUploader', () => {
...
+
+    await act(async () => {
+      jest.runAllTimers()
+    })
+
+    expect($$(`.adm-image-img`).length).toBe(4)
+    expect($$(`.adm-image-img`)[1]).toHaveAttribute('src', 'one.png')
+    expect($$(`.adm-image-img`)[2]).toHaveAttribute('src', 'two.png')
+    expect($$(`.adm-image-img`)[3]).toHaveAttribute('src', 'three.png')
+    expect(container).toMatchSnapshot()
```
Comment (miracles1919) [inline src/components/image-uploader/tests/image-uploader.test.tsx:211]: 上面 expect 了顺序，应该不用再 expect snap了

Comment (heiyu4585) [reply src/components/image-uploader/tests/image-uploader.test.tsx]: done

**Review by miracles1919 — APPROVED** [review]

**Comment by zombieJ** [comment]
这样会导致所有上传完成才能更新 value，不过似乎也没有更好方式。毕竟要保持顺序...

**Pull request merged**